frontend/src/app/template/template-builder/components/team-builder-dialog/team-builder-dialog.component.ts

```diff
@@ -163,6 +182,16 @@ export class TeamBuilderDialogComponent {
     } as ApiTeam;
   }
 
+  onMemberFilterChanged(filter: string){
+    console.log(`MEMBER: was: ${this.memberFilterText} is: ${filter}`)
+    this.memberFilterText = filter;
```

Comment:
do you want to keep this or was this for debugging?

Reply:
Thanks, this was for debugging and has been removed.